qiskit/ignis/verification/randomized_benchmarking/fitters.py

```diff
@@ -1140,3 +1140,275 @@ def plot_rb_data(self, pattern_index=0, ax=None,
 
         if show_plt:
             plt.show()
+
+
+class NonCliffordRBFitter(RBFitterBase):
```

Comment:
Should this be CNOTDihedralRBFitter? There could be other types of non-clifford rb

Reply:
You are correct. I have refactored 'NonClifford' / 'nonclifford' --> 'CNOTDihedral' / 'cnotdihedral' throughout the entire code (including the tests and tutorial)